.github/workflows/cicd.yml

```diff
@@ -4,12 +4,16 @@ name: CI/CD
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v2
       - uses: ./
         with:
-          nix_archives_url: https://github.com/nixbuild/nix-quick-install-action/releases/download/v2
+          # FIXME: nix_archives_url: https://github.com/nixbuild/nix-quick-install-action/releases/download/v2
+          nix_archives_url: https://stephank.nl/dump/nix
```

Comment:
I'd say this is OK for bootstrapping. I did a similar thing for v1, I created a v0 release and uploaded the archives manually to it.

Reply:
How would you like to go about this? I guess either a v3pre or reuse v0?

There's now a full set of binaries from the CI run on my fork we can probably just copy over: https://github.com/stephank/nix-quick-install-action/releases/tag/v2

Reply:
I'll think reusing v0 is the best, since we don't have any pre-releasing workflow in place. Did you copy those binaries to https://github.com/stephank/nix-quick-install-action/releases/tag/v2 manually? I just wonder since the release itself is based on an older revision.

Reply:
No, that release was generated by a CI run on my fork: https://github.com/stephank/nix-quick-install-action/runs/1170679961

Do you mean the Nix 3.0pre revision used? I didn't change that from master.

Reply:
To clarify, the binaries on stephank.nl were produced locally on my Mac, which is on nixFlakes in nixos-unstable, and a NixOS vm builder for Linux. Those are then used by CI to build the ones found in the release.

I had a little trouble finding the correct `nix build` invocation for the local builds, and ended up using flakes-compat temporarily.

Reply:
Hmm. So I found the action that created the `v2` release on your repo: https://github.com/stephank/nix-quick-install-action/runs/1170679961?check_suite_focus=true. You can clearly see that a release was created from the log:

```
New release:  -> v2
https://github.com/stephank/nix-quick-install-action/releases/tag/v2
```

This action ran for your MacOS commit, https://github.com/stephank/nix-quick-install-action/commit/48adfc468072636d3980a8fd1268d10c009bfc29. So everything is fine. But if you go to https://github.com/stephank/nix-quick-install-action/releases/tag/v2 it points you to and older revision (the `v2` tagged by me). So it seems the tag `v2` was not updated in your repo, but a release was still created. And the GitHub UI makes it look like the release was made for that older commit, which is not true.

Oh well, this doesn't really matter for us. I will update my `v0` release with the binaries from your `v2` release, then you can update your PR with the `v0` bootstrapping url.

Reply:
You can use this url for bootstrapping now: https://github.com/nixbuild/nix-quick-install-action/releases/download/v0

Reply:
Ah, sorry! That is confusing. I didn't consider what git rev the tag would actually point to.

The PR is updated. I also added 9a85ad1, because the `pull_request` CI run was failing, while the `push` CI run in my fork was succeeding. (Curiously, it was already running `pull_request` CI before I merged your change in ac7e1b2, despite the workflow here not indicating that event.)